src/PiWeb.Import.Sdk/Modules/ImportFormat/FileSourceExtensions.cs

```diff
@@ -0,0 +1,77 @@
+﻿#region copyright
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+/* Carl Zeiss Industrielle Messtechnik GmbH        */
+/* Softwaresystem PiWeb                            */
+/* (c) Carl Zeiss 2024                             */
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#endregion
+
+namespace Zeiss.PiWeb.Import.Sdk.Modules.ImportFormat;
+
+#region usings
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+#endregion
+
+/// <summary>
+/// Provides methods to work with <see cref="IFileSource"/> more easily.
+/// </summary>
+public static class FileSourceExtensions
+{
+    #region methods
+
+    /// <summary>
+    /// Checks if <paramref name="fileSource"/> has the requested file extension <paramref name="extension"/>.
+    /// </summary>
+    /// <param name="fileSource">The <see cref="IFileSource"/> to check.</param>
+    /// <param name="extension">The file extension (including the period ".").</param>
+    /// <returns><c>true</c> if the file has the extension; otherwise <c>false</c>.</returns>
+    public static bool HasExtension(this IFileSource fileSource, string extension)
+    {
+        if (!extension.StartsWith('.'))
+            extension = '.' + extension;
+
+        return fileSource.Name.EndsWith(extension, StringComparison.OrdinalIgnoreCase);
+    }
+
+    /// <summary>
+    /// Checks if <paramref name="fileSource"/> has any of the requested file extension <paramref name="extension"/>.
+    /// </summary>
+    /// <param name="fileSource">The <see cref="IFileSource"/> to check.</param>
+    /// <param name="extension">The file extensions (including the period ".").</param>
+    /// <returns><c>true</c> if the file has any of  the extensions; otherwise <c>false</c>.</returns>
+    public static bool HasAnyExtension(this IFileSource fileSource, IEnumerable<string> extension)
+    {
+        return extension.Any(f => f.Equals(fileSource.Extension, StringComparison.OrdinalIgnoreCase));
```

Comment:
The parameter must be named "extensions".
And the check should use HasExtension() instead of an Equals() on the fileSource.Extension:
`return extensions.Any(extension => fileSource.HasExtension(extension));`